contracts/LendingPool.sol

```diff
@@ -72,7 +72,10 @@ contract LendingPool is OwnableUpgradeable, ERC721Upgradeable, Clone {
         if(toReduce > currentDailyBorrows){
             currentDailyBorrows = toAdd;
         } else {
-            currentDailyBorrows = uint216(currentDailyBorrows - toReduce) + toAdd;
+            unchecked {
+                // Resulting daily borrow has to be over 10^47 ETH
+                currentDailyBorrows = uint216(currentDailyBorrows - toReduce) + toAdd;
```

Comment:
uint216(currentDailyBorrows - toReduce) can be unchecked but toAdd could overflow

Reply:
https://github.com/LlamaLend/contracts/pull/7/commits/d395180155a6634e7a3ab6d6074f3c11e71ccb05